doc/source/visualizing/FITSImageData.ipynb

```diff
@@ -79,7 +79,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "which took the same parameters as `ProjectionPlot` except the width, because `FITSProjection` and `FITSSlice` always make slices and projections of the width of the domain size, at the finest resolution available in the simulation, in a unit determined to be appropriate for the physical size of the dataset."
```

Comment:
I think the text you deleted is fine here, but I also think that you could maybe add a quick example to this file of making the same projection with a different width, which would be clearer.

Reply:
Yes, I added a quick example, set a new pull request.